test/functional/master/test_build_cancellation.py

```diff
@@ -0,0 +1,66 @@
+import os
+import subprocess
+import tempfile
+import time
+from unittest import skipIf
+import yaml
+
+from app.util.process_utils import is_windows
+from test.framework.functional.base_functional_test_case import BaseFunctionalTestCase
+from test.functional.job_configs import JOB_WITH_SLEEPS_90SEC, JOB_WITH_SLEEPING_ATOMIZER_90SEC
+
+
+@skipIf(is_windows(), 'Fails on AppVeyor; see issue #345')
```

Comment:
I intentionally kept the "skip" as the test is very similar to the one I copied from. But lets discuss this. Same comment for the not having `job config` for Windows as I was planning to skip tests on Windows.